src/Runner.jl

```diff
@@ -331,10 +331,6 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
             append!(flags, [
                 # Find GCC toolchain here (for things like libgcc_s)
                 "--gcc-toolchain=/opt/$(aatriplet(p))"
-                # Use libgcc as the C runtime library
-                "-rtlib=libgcc"
-                # Use libstdc++ as the C++ runtime library
-                "-stdlib=libstdc++"
```

Comment:
The comments above reference `-rtlib` and `-stdlib` which now don't make much sense.  Should rearrange them as well.